cli/flags.rs

```diff
@@ -340,6 +340,15 @@ Automatically downloads Prettier dependencies on first run.
 
   deno fmt myfile1.ts myfile2.ts",
         )
+        .arg(
+          Arg::with_name("prettierrc")
```

Comment:
Conflict with the global `config`, so temporarily named `prettierrc`

Maybe `preset` is a better option?